app/models/concerns/sample_sheet.rb

```diff
@@ -4,8 +4,30 @@
 # Namespace for helper modules to assist ActiveRecord classes in rendering
 # sample sheets
 module SampleSheet
+  # Provides run helper methods for sample sheet generation
+  module Run
+    # Returns a list of wells associated with the plate in column order
+    # Example: [<Well position:'A1'>, <Well position:'A2'>, <Well position:'B1'>]) =>
+    #          [<Well position:'A1'>, <Well position:'B1'>, <Well position:'A2'>]
+    def sorted_wells
+      wells.sort_by { |well| [well.column.to_i, well.row] }
+    end
+  end
+
   # Provides well helper methods for sample sheet generation
   module Well
+    # The sequencing_kit_box_barcode of this plate if well belongs to plate 1
+    # used for 2-plate sample sheets
+    def sequencing_kit_box_barcode_plate_1
+      plate.sequencing_kit_box_barcode if plate.plate_number == 1
+    end
+
+    # The sequencing_kit_box_barcode of this plate if well belongs to plate 2
+    # used for 2-plate sample sheets
+    def sequencing_kit_box_barcode_plate_2
```

Comment:
might be worth waiting until we merge develop but maybe we could have a method with an argument in the configuration. That would make it much more dynamic. Then for plate 1 and 2 we could have the same method but with a different argument?